tests/components/zwave_js/test_sensor.py

```diff
@@ -36,3 +45,31 @@ async def test_energy_sensors(hass, hank_binary_switch, integration):
     assert state.state == "0.16"
     assert state.attributes["unit_of_measurement"] == ENERGY_KILO_WATT_HOUR
     assert state.attributes["device_class"] == DEVICE_CLASS_ENERGY
+
+
+async def test_disabled_notification_sensor(hass, multisensor_6, integration):
+    """Test sensor is created from Notification CC and is disabled."""
+    ent_reg = await async_get_registry(hass)
+    entity_entry = ent_reg.async_get(NOTIFICATION_MOTION_SENSOR)
+
+    assert entity_entry
+    assert entity_entry.disabled
+    assert entity_entry.disabled_by == DISABLED_INTEGRATION
+
+    # Test enabling entity
+    updated_entry = ent_reg.async_update_entity(
+        entity_entry.entity_id, **{"disabled_by": None}
+    )
+    await hass.async_block_till_done()
+    assert updated_entry != entity_entry
+    assert updated_entry.disabled is False
+
+    # the part below needs to be fixed
+    # the entity state is not available yet after enabling the entity
+    await hass.config_entries.async_reload(integration.entry_id)
+    await hass.async_block_till_done()
+    await hass.async_block_till_done()
```

Comment:
Do we really need two block till done?

Reply:
Didn’t try without, kind of guessed it was needed. Feel free to remove, need to get some sleep now haha

Reply:
I pulled down the repo and tested the changes successfully. I'll go ahead and remove these lines on Marcel's behalf